customer_service/lib/FileManager/File/PersistFile.php

```diff
@@ -0,0 +1,160 @@
+<?php
+namespace Lib\FileManager\File;
+
+use Lib\FileManager\File\File;
+
+class PersistFile implements File
+{
+    private $id;
+    private $name;
+    private $fileGroupId;
+    private $fileTypeId;
+    private $fileTypeName;
+    private $oldName;
+    private $extension;
+    private $path;
+    private $size;
+    private $createdTime;
+    private $lastUpdate;
+    private $status;
+
+    public function __construct($id, $fileGroupId, $fileTypeId, $fileTypeName, $oldName, $extension, $path, $size, $createdTime, $lastUpdateTime, $status)
+    {
+        $this->id = $id;
+        $this->name = $id;
+        $this->fileGroupId = $fileGroupId;
+        $this->fileTypeId = $fileTypeId;
+        $this->fileTypeName = $fileTypeName;
+        $this->oldName = $oldName;
+        $this->extension = $extension;
+        $this->path = $path;
+        $this->size = $size;
+        $this->createdTime = $createdTime;
+        $this->lastUpdateTime = $lastUpdateTime;
+        $this->status = $status;
+    }
+
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    public function setId($id)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/gh/BuildForSDG/Team-064-Backend/pullRequest?prid=5616370)